docs/user-guide/using-a-zos-attributes-file.md

```diff
@@ -0,0 +1,68 @@
+# Using a z/OS attributes file
+
+Use a `.zosattributes` file to control how files are converted and tagged when you upload a local directory to a USS directory on the mainframe.
+
+## What is a `.zosattributes` file
+
+A `.zosattributes` file saved in your source directory provides uploading and downloading instructions to the mainframe. A line in the file can specify a code comment or instructions.
+
+An example `.zosattributes` file:
+
+```json
+// highlight-start
+# pattern local-encoding remote-encoding
+# Don't upload the node_modules directory
+// highlight-end
+node_modules -
+// highlight-start
+# Don't upload files that start with periods
+// highlight-end
+.* -
+// highlight-start
+# Upload jpg images in binary
+// highlight-end
+*.jpg binary binary
+// highlight-start
+# Convert CICS Node.js profiles to EBCDIC
+// highlight-end
+*.profile ISO8859-1 EBCDIC
+```
```

Comment:
Wondering if we should just use a `shell` like highlighter for this block of code.
| Before | After |
| - | - |
| <img width="694" alt="image" src="https://github.com/user-attachments/assets/dc560ff2-3b09-4e28-b4e0-c6fc1dd98a21" /> | <img width="409" alt="image" src="https://github.com/user-attachments/assets/edfa6538-ab2b-4095-8c1d-a7cbb2a830f1" /> |

Note: The `after` is just how GitHub displays _"```shell"_. Not sure if it will render the same way in the live site :yum: